--- ./server.rb.bak       2023-11-15 11:16:20.000000000 -0500
+++ ./server.rb   2024-09-19 00:11:45.920565280 -0400
@@ -356,9 +356,9 @@
                   xml.disk(:type => "file", :device => "cdrom") do
                     xml.driver(:name => "qemu", :type => "raw")
                     xml.source(:file => "#{iso_dir}/#{iso_file}")
-                    xml.target(:dev => "hdc", :bus => "ide")
+                    xml.target(:dev => "sda", :bus => "scsi")
                     xml.readonly
-                    xml.address(:type => "drive", :controller => 0, :bus => 1, :unit => 0)
+                    xml.address(:type => "drive", :controller => 0, :bus => 0, :unit => 0)
                   end
                 end
